Dodo/Dodo/ViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController.swift
+//  Dodo
+//
+//  Created by Blake Andrew Price on 10/17/19.
+//  Copyright © 2019 Blake Andrew Price. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
Did you mean to name this "ViewController"? You should think about a more descriptive name.